jail-buddy.lic

```diff
@@ -54,7 +54,12 @@ def get_out_of_jail(old_loc)
   Script.running.find_all { |s| !s.paused? && !s.no_pause_all }.each(&:pause)
   stop_script('go2') if Script.running?('go2')
   echo('** DON\'T PANIC JAIL BUDDY HAS YOUR BACK! **')
+  fput('stand') if not standing?
```

Comment:
:x: There's a [helper in common.lic](https://github.com/rpherbig/dr-scripts/blob/master/common.lic#L260) that will do this (note you'd call it with the prefix, e.g. `DRC.fix_standing`).